### ISSUE TITLE
Document the process title module (require('title')

### ISSUE THREAD

**Issue by kostja** [tarantool/doc#179]
Here's the module implementation: 
https://github.com/tarantool/tarantool/blob/1.7/src/lua/title.lua

Tests seem to be missing :(

**Comment by lenkis** [comment]
Should "custom_proc_title" parameter be deprecated?

**Comment by lenkis** [comment]
Module "title" is not yet ready to be documented/recommended for public use. This task postponed.

**Comment by lenkis** [comment]
@kostja please confirm the "postponed" status above and decide on the module's future (improve/deprecate).

**Comment by lenkis** [comment]
@rtsisyk please explain what's wrong with the title module (implemented in https://github.com/tarantool/tarantool/issues/475).

**Comment by rtsisyk** [comment]
I don't think that we need to document this internal stuff. Can I close this ticket?

**Comment by kostja** [comment]
@rtsisyk I had a customer request asking for features implemented in this module. It's OK if you think that this module is not ready for publishing, but then please say what exactly is wrong with it, I don't see it from the exchange above.

**Comment by rtsisyk** [comment]
https://github.com/tarantool/tarantool/issues/2252#issuecomment-296944444

**Comment by lenkis** [comment]
blocked by API changes under https://github.com/tarantool/tarantool/issues/2252

**Comment by patiencedaur** [comment]
Looks like this is not a priority for development. Closing as stale.
